R/build-home-index.R

```diff
@@ -21,6 +21,7 @@ build_home_index <- function(pkg = ".", quiet = TRUE) {
       data$index <- markdown(src_path)
       render_page(pkg, "home", data, "index.html")
     } else if (file_ext == "Rmd") {
+      data$index <- "$body$"
```

Comment:
This needs to `index` since that's what it is called in the templates,including the default template.

The default template could be changed to `content`, though that's an API change and would break any custom home templates that are already out there.